website/addons/s3/settings/defaults.py

```diff
@@ -4,6 +4,22 @@
 
 ALLOWED_ORIGIN = '*'
 
+BUCKET_LOCATIONS = {
```

Comment:
It might make sense to use a JSON file rather than a python variable for this.

- `BUCKET_LOCATIONS` isn't meant to be configured; it's just a constant
- Using JSON would allow this data to be loaded in both Python and JS without duplication or contextVars hackery.